### PR TITLE
Setting components

### PR DESCRIPTION
**Pull request by BonkeGcobo**
1) Set up the components for the BookStore App
2) Add react-router to navigate to different pages.

### CODE REVIEW

File: src/components/Books/Book.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+const Book = (props) => {
+  const { name, author } = props;
+
+  return (
+    <div className="book">
+      {name}
+      {author}
+    </div>
+  );
```
Comment (micheaol) [inline src/components/Books/Book.js]: Please add a button for removing books to this component according to the project requirement. The button wouldn't remove the books yet, this is just a placeholder for now.

File: src/components/Books/AddBook.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  <div className="add-book">
+    <h1>Add Book</h1>
+    <form className="form">
+      <input type="text" placeholder="Enter Book Name" />
+      <input type="submit" />
+      <br />
+      <br />
+      <input type="text" placeholder="Enter Book Name" />
+      <input type="submit" />
+    </form>
```
Comment (micheaol) [inline src/components/Books/AddBook.js]: Your form for creating a `new book` should have only one `submit button` according to the project requirement.

File: src/components/Category/Category.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+function Category() {
+  return (
+    <div>
+      <h1>Coming Soon</h1>
+    </div>
```
Comment (micheaol) [inline src/components/Category/Category.js]: Please ensure that the `Category component` displays only a button with text "Check status according to the project requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @BonkeGcobo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by andersonlebon — APPROVED** [review]
## Hi @BonkeGcobo 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**